src/java_bytecode/java_object_factory.cpp

```diff
@@ -119,8 +139,10 @@ class java_object_factoryt
     irep_idt class_identifier,
     bool skip_classid,
     allocation_typet alloc_type,
-    bool override,
+    bool override_,
```

Comment:
Because `override` is a keyword in C++ 11, and while the compiler is kind enough to compile this, I don't think we should rely on that. Same story with the `final()` method in `languaget` and friends.